src/control/lib/daos/system_prop_test.go

```diff
@@ -177,3 +181,67 @@ func TestDaos_SystemPropertyKeys(t *testing.T) {
 		}
 	}
 }
+
+func TestDaos_parseDuration(t *testing.T) {
+	for name, tc := range map[string]struct {
+		input  string
+		expDur time.Duration
+		expErr error
+	}{
+		"invalid unit": {
+			input:  "40q",
+			expErr: errors.New("unknown unit"),
+		},
+		"missing number": {
+			input:  "m",
+			expErr: errors.New("invalid"),
+		},
+		"invalid number": {
+			input:  "onem",
+			expErr: errors.New("invalid"),
+		},
+		"empty string": {
+			input:  "",
+			expDur: 0,
+		},
+		"no unit": {
+			input:  "300",
+			expDur: 5 * time.Minute,
+		},
+		"milliseconds": {
+			input:  "3000ms",
+			expDur: 3 * time.Second,
+		},
```

Comment:
Incorrect result! The code is only parsing out the `s` not the `ms`

Reply:
That result is from `time.ParseDuration()`... 3000ms is 3 seconds, no?

Reply:
Ah wait, you're right.